tests/Feature/Service/Student/UpdateStudentProjectServiceTest.php

```diff
@@ -49,7 +49,7 @@ public function testCanUpdateStudentProjectByProjectId(): void
         $this->assertEquals('Project One', $updatedProject->name);
```

Comment:
Hi Bego, I executed the feature tests and I found that this assertion doesn't pass the test, I think that $updatedProject->name is keeping the factory one(Lebsack-Witting) instead of the  $data['name'](Project One):
![image](https://github.com/user-attachments/assets/ccfa32f8-7492-466d-8fe0-a3c2e1f29b62)

Reply:
It passes ok for me:
![image](https://github.com/user-attachments/assets/504b579b-fd82-4bd0-bb95-5e0a5394cba0)